src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_specific_price.html.twig

```diff
@@ -36,7 +36,7 @@
   {% set column_default_xl_3 = 'col-xl-4' %}
 {% endif %}
 
-<div class="card card-block">
+<div class="card card-body">
```

Comment:
Same question here, is it normal that card and card-body are the same div?

Reply:
And they have no header nor footer?

Reply:
It should be ok. Card does not have to contain header nor footer. :)

Reply:
Header and footer are optional tbh

Reply:
In previous versions of bootstrap I saw examples with `<div class="card card-body>` because `card-body` only adds a pading, but I have seen in the last version that they keep it separate. https://getbootstrap.com/docs/5.1/components/card/#body
I think it is better to have it separated, since it makes each `div` have a unique responsibility.